app/serializers/version_serializer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class VersionSerializer < ActiveModel::Serializer
-  attributes :number, :published_at
+  attributes :number, :published_at, :spdx_expression
```

Comment:
Should we also include `original_license` here?

Reply:
My reason against was that it could hugely impact the size of versions over the wire. Pypi in particular has a lot of raw licenses copied and pasted.

Reply:
How do we handle that for project licenses?

Reply:
We don't really but at the moment you're just getting one license field per project call, with this change you'll be getting one per version

Reply:
But if we don't return the original license, then we can do nothing for non-spdx licenses. There is a case for some of those that we want to handle (for example, vcr is now licensed under the Hippocratic license which can't be represented by spdx at present)